rom/dev/tests/rom_integration_tests/test_idevid_derivation.rs

```diff
@@ -54,3 +67,48 @@ fn test_idev_subj_key_id_algo() {
             .unwrap();
     }
 }
+
+fn fuses_with_random_uds() -> Fuses {
+    const UDS_LEN: usize = core::mem::size_of::<u32>() * 12;
+    let mut uds_bytes = [0; UDS_LEN];
+    rand_bytes(&mut uds_bytes).unwrap();
+    let mut uds_seed = [0u32; 12];
+
+    for (word, bytes) in uds_seed.iter_mut().zip(uds_bytes.chunks_exact(4)) {
+        *word = u32::from_be_bytes(bytes.try_into().unwrap());
+    }
+    Fuses {
+        uds_seed,
+        ..Default::default()
+    }
+}
+
+#[test]
+#[cfg_attr(not(feature = "slow_tests"), ignore)]
+fn test_generate_csr_stress() {
+    const NUM_TESTS: usize = 1000;
+
+    for _ in 0..NUM_TESTS {
+        let fuses = fuses_with_random_uds();
```

Comment:
It would be nice to make sure the test works on a single iteration to avoid regressions in the test code. And then the slow test would just run that `NUM_TESTS` times.

Could even just conditionally choose NUM_TESTS based on the config value.

Reply:
Good idea. Updated.